build-scripts/docker/build-connector-image.sh

```diff
@@ -40,9 +40,9 @@ function main() {
   # Tag the image with a commit hash if we provide it
   if [[ -z "$DOCKER_TAG" ]];
   then
-    IMAGE_TAGS="-t $IMAGE_NAME:latest"
+    IMAGE_TAGS="-t $IMAGE_NAME:latest -t $IMAGE_NAME:dev"
```

Comment:
Why do we add `dev` tag here and, at the same time, change in all tests `dev` to `latest`?

Reply:
Right, I will remove that again